settings/Core.setting.php

```diff
@@ -1179,7 +1195,11 @@
     'is_domain' => 1,
     'is_contact' => 0,
     'description' => ts('Theme to use on frontend pages'),
-    'help_text' => NULL,
+    'help_text' => ts('
+        The theme system allows you to change CiviCRM\'s appearance by replacing important CSS files.
+        On WordPress, Joomla, or a similar CMS, the frontend theme determines the appearance on user-facing screens, such as the "Event Registration" screen.
```

Comment:
newlines and space-formatting is discouraged in ts - https://docs.civicrm.org/dev/en/latest/translation/#avoid-multi-line-strings

I know there's some pre-existing already but since the additions here will require retranslation might as well fix.

Reply:
thanks @demeritcowboy - should be fixed now